crt_portal/cts_forms/migrations/0093_auto_20201218_1557.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.17 on 2020-12-18 20:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0092_profile'),
+    ]
+
+    operations = [
+        migrations.RunSQL("""
+            CREATE OR REPLACE VIEW email_report_count AS
```

Comment:
Would it be easier to select `cts_forms_report.*` here instead of only ID/count and then joining them again with the columns from report?

Reply:
I went back and forth on this. The thing I was trying to avoid was having the `EmailReportCount` model duplicate all of the `Report` fields, because there are a lot of them. I thought about having `EmailReportCount` inherit from `Report`, but that seemed awkward. Do you have thoughts on this?

Reply:
Sounds good to me, I think this is a reasonable trade-off for now! Thanks for considering different paths.

The Inheriting doesn't strike me as too awkward since I _think_ we can get by with something as small as:
```python
class EmailReportCount(Report):
    """
    Model for view created in `0093_create_email_report_count_view` where we append
    aggregate counts of `contact_email` to each `Report` record
    """  
    email_count = models.IntegerField()

    class Meta:
        managed = False
        db_table = 'email_report_count'
```